src/tekore/_client/api/playlist/items.py

```diff
@@ -139,3 +141,69 @@ def playlist_remove(
         return self._generic_playlist_remove(
             playlist_id, {"tracks": items}, snapshot_id
         )
+
+    @scopes([scope.playlist_modify_public], [scope.playlist_modify_private])
+    @send_and_process(top_item("snapshot_id"))
+    def playlist_remove_occurrences(
+        self,
+        playlist_id: str,
+        refs: List[Tuple[str, int]],
+        snapshot_id: Optional[str] = None,
+    ) -> str:
+        """
+        Remove items by URI and position.
+
+        .. warning::
+        This feature is undocumented and may stop working at any time.
+
+        Parameters
+        ----------
+        playlist_id
+            playlist ID
+        refs
+            a list of tuples containing the URI and index of items to remove
+        snapshot_id
+            snapshot ID for the playlist
+        Returns
+        -------
+        str
+            snapshot ID for the playlist
+        """
+        gathered = {}
```

Comment:
IMO this method could be simpler:

```python
    def playlist_remove_occurrences(
        self, playlist_id: str, refs: dict[str, list[int]], snapshot_id: Optional[str] = None
    ) -> str:
        tracks = [{"uri": uri, "positions": indices} for uri, indices in refs.items()]
        return self._generic_playlist_remove(
            playlist_id, {"tracks": tracks}, snapshot_id
        )
```

This of course changes the signature, but maybe that is not an issue, given that it probably has never been used by anyone?

Reply:
I think I might have a clue as to why `refs` was made a `list` instead of a `dict`: it could be because of chunking.  I have a use case where I need to remove large number of tracks from a playlist, and that is currently failing with HTTP error 413, because this method does not support chunking (it does not have a `@chunked` decorator). I tried adding the `@chunked` decorator to the method, but that does not seem to work with `ref` being a dictionary.

Reply:
So I tested this with chunking, and the way it works is not like the rest of the endpoints. If you want to use chunking with this one, you _must_ provide a `snapshot_id`, so all chunks will be executed against the same version of a playlist. It does make sense when I think of it: if you've removed a bunch of positions in the first chunk, the next chunk's positions would not match anymore.

I am not sure how to implement this. We could do something like this I suppose

```
if len(refs) > chunk_size and snapshot_id is None:
    raise RuntimeError("For chunked requests snapshot_id must not be None")
```

The tricky part is that we do not have access to `chunk_size` in the method itself, so this logic needs to go into the `chunked` decorator implementation. What do you think?